tf2_tools/tf2_tools/view_frames.py

```diff
@@ -50,12 +51,19 @@ def main():
         '--wait-time', '-t', type=float, default=5.0,
         help='Listen to the /tf topic for this many seconds before rendering the frame tree')
     parser.add_argument('-o', '--output', help='Output filename')
+    parser.add_argument('--tf_topic', help='Topic for dynamic transforms',
```

Comment:
Could also be `tf_dynamic_topic`. Open to either given the help text next to it says dynamic